doc/base/classes.xml

```diff
@@ -22390,7 +22390,19 @@
 			<argument index="0" name="rel_vec" type="Vector3">
 			</argument>
 			<description>
-				Move the body in the given direction, stopping if there is an obstacle. The returned vector is how much movement was remaining before being stopped.
+				Move the body in the given direction, stopping if there is an obstacle. If as a result of a movement there will be any collision then informations about this collision will be in returned dictionary.
+				Dictionary will contains those keys:
+				"position" - collision position
```

Comment:
I see, was wondering how to deal with that, but checked some other place where dictionary keys were explained and because of that I though there is no better way. Will fix later!